submissions/SphericalCat/hooli-popup/index.html

```diff
@@ -0,0 +1,270 @@
+                                <a class="popup_link link" href="#">
+                                    <div class="contacts items_img"></div>
+                                    Contacts
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link link" href="#">
+                                    <div class="videos items_img"></div>
+                                    Videos
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link link" href="#">
+                                    <div class="photo items_img"></div>
+                                    Photo
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link link" href="#">
+                                    <div class="maps items_img"></div>
+                                    Maps
+                                </a>
+                            </li>
+                        </ul>
+                        <input type="checkbox" id="check_more" class="check_more">
+                        <label class="see_more" for="check_more">
+                            <span class="see_more_item">Hug the kitten</span>
+                        </label>
+                        <div class="popup_two">
+                            <a href="#"><img class="hug_cat" src="styles/img/logo.png" alt="cat" width="230"></a>
+                            <ul class="popup_two_items">
+                                <li>
+                                    <a class="link" href="#">
+                                        <div class="food items_img"></div>
+                                        Food
+                                    </a>
+                                </li>
+                                <li>
+                                    <a class="link" href="#">
+                                        <div class="game items_img"></div>
+                                        Game
+                                    </a>
+                                </li>
+                                <li>
+                                    <a class="link" href="#">
+                                        <div class="experience items_img"></div>
+                                        Experience
+                                    </a>
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                </li>
+                <li class="my_icon menu_item"><a href="https://github.com/SphericalCat/kottans-frontend">
+                        <img src="styles/img/pngegg1.png" alt="i" width="50"></a>
```

Comment:
For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty), and if you even use `alt`, use naming that brings sense to image